spec/models/specification_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe Specification, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Cruft file
Be sure to use `git add -p` to add your files so you can crosscheck them before commiting